content/en/docs/project-user-guide/application-workloads/services.md

```diff
@@ -1,10 +1,167 @@
+
+{{< notice tip>}}
+
+In Kubesphere, the **Virtual IP** is a default type of a service that is created by **stateless** or **stateful**.<br />
+
+If you want to create a Headless service, you should use the mode of **YAML**.
+
+{{</ notice>}}
+
+## Prerequisites
+
+- You need to create a workspace, project and `project-regular` account. Please refer to the [Getting Started with Multi-tenant Management](../../../quick-start/create-workspace-and-project) if not yet.
+- You need to sign in with `project-admin` account and invite `project-regular` to enter the corresponding project if not yet. Please refer to [Invite Member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+## How to create a Service
+
+![create_service_type](/images/docs/project-user-guide/workloads/create_service_type.png)
+
+Create a service that has these methods in the Kubesphere: **Stateless Service**, **Stateful Service**, and **External Service**. what's more, you can custom creation through **Specify Workloads** and **Edit by YAML**.
+
+There are some ways to create a service
+
+- **Stateless Service**
+
+  Create a Stateless Service also create a **Deployment** workload.
+
+  For more information **Stateless**, please visit [Deployment](../../application-workloads/deployments)
+
+- **Stateful Service**
+
+  Create a Stateless Service also create a **Statefulset** workload.
+
+  For more information **Statefulset**, please visit [Statefulset](../../application-workloads/statefulsets)
+
+- **External Service**
+
+  The service created by External Service is a service internet access method. It is different from Stateless and stateful.
+
+- **Specify Workloads**
+
+  Create a service with existing Pods.
+
+- **Edit by YAML**
+
+  Create directly with Yaml, you can upload and download Yaml configuration files
+
+{{< notice tip>}}
+
+  The value of `annotations:kubesphere.io/serviceType` keywords is one of them **statelessservice** , **statefulservice** , **externalservice** and **None**.
+
+{{</ notice>}}
+
+## Create a **Stateless Service**
+
+Into **Application Workload**, click **Services**
+
+### Step 1: click the **Create**, choose **Stateless Service**
+
+![services_lists](/images/docs/project-user-guide/workloads/services_lists.png)
+
+![stateless_form](/images/docs/project-user-guide/workloads/stateless_form.png)
+
+### Step 2: Input Base data
+
+![stateless_form_1](/images/docs/project-user-guide/workloads/stateless_form_1.png)
+
+- **Name**: The name of the Service and Deployment, which is also the unique identifier.
+- **Alias**: The alias name of the service, making resources easier to identify.
+- **Version**: It can only contain lowercase letters and numbers. The maximum length of characters is set to 16.
+
+{{< notice tip >}}
+
+**Name** have named two configs, `Deployment` and `Service`.
```

Comment:
Sorry, I don't quite understand this sentence...I guess you mean the value of `kind` in YAML? Two configurations in the yaml file: one `Deployment` and the other one is `Service`, both of which contain the value of **Name**. If so, please change it to:

The value of **Name** is used in both configurations, one for `Deployment` and the other for `Service`.

Reply:
done